packages/ibm-products-styles/src/components/TagOverflow/_tag-overflow.scss

```diff
@@ -63,59 +62,58 @@ $block-class-modal: #{$block-class}-modal;
   max-inline-size: $spacing-09;
 }
 
-.#{c4p-settings.$carbon-prefix}--popover
-  .#{c4p-settings.$carbon-prefix}--popover-content {
+.#{$carbon-prefix}--popover .#{$carbon-prefix}--popover-content {
   white-space: normal;
 }
 
-.#{$block-class-overflow} {
+.#{$block-class-popover} {
   display: inline-block;
   vertical-align: bottom;
-  .#{c4p-settings.$carbon-prefix}--tag.#{c4p-settings.$carbon-prefix}--tag--interactive {
+
+  .#{$carbon-prefix}--tag.#{$carbon-prefix}--tag--interactive {
     border: 0;
   }
 
-  .#{c4p-settings.$carbon-prefix}--popover
-    .#{c4p-settings.$carbon-prefix}--popover-content {
+  .#{$carbon-prefix}--popover .#{$carbon-prefix}--popover-content {
     padding: $spacing-05;
   }
 }
 
-.#{$block-class-overflow}--hidden {
+.#{$block-class-popover}--hidden {
   overflow: hidden;
   max-inline-size: 0;
   visibility: hidden;
 }
 
-@include mixins.block-wrap('#{$block-class-overflow}__el') {
-  &.#{$block-class-overflow}__el {
+@include mixins.block-wrap('#{$block-class-popover}__el') {
+  &.#{$block-class-popover}__el {
     // removes the min width in Carbon
     min-inline-size: initial;
     text-align: start;
   }
 
-  .#{$block-class-overflow}__trigger {
+  .#{$block-class-popover}__trigger {
     /* stylelint-disable-next-line declaration-no-important */
     border: none !important;
     font-family: inherit;
   }
 
-  .#{c4p-settings.$carbon-prefix}--link:active,
-  .#{c4p-settings.$carbon-prefix}--link:active:visited,
-  .#{c4p-settings.$carbon-prefix}--link:active:visited:hover {
+  .#{$carbon-prefix}--link:active,
+  .#{$carbon-prefix}--link:active:visited,
+  .#{$carbon-prefix}--link:active:visited:hover {
     color: $text-inverse;
   }
 
-  .#{$block-class-overflow}__tag-list {
+  .#{$block-class-popover}__tag-list {
     display: flex;
     flex-direction: column;
   }
 
-  .#{$block-class-overflow}__show-all-tags-link {
+  .#{$block-class-popover}__show-all-tags-link {
     color: $link-inverse;
-    margin-block-start: $spacing-03;
 
-    &:hover {
+    &:hover,
+    &:visited:hover {
```

Comment:
This fixes the "View all tags" hover color issue.

`&:visited:hover` is req'd to override the baseline `cds--link:visited:hover`.